### PR TITLE
AAP-32141 Add Installing on Operator chapter to RHDH install guide

### PR DESCRIPTION
**Pull request by ariordan-redhat**
AAP-32141 Add Installing on Operator chapter to RHDH install guide

Affects titles/aap-plugin-rhdh-install/

Preview here: https://ariordan-redhat.github.io/aap-builds/aap-plugin-rhdh-install-scaffold-plugin-operator-install-2024-09-26.html

### CODE REVIEW

File: downstream/modules/devtools/proc-rhdh-backup-operator-configmap.adoc [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+When you installed the {RHDHShort} Operator, a namespace was created for it.
+Select *Topology* and look for the {RHDHShort} Operator in the *Project* dropdown list.
+The default namespace is `rhdh-operator`.
+. Run the following command to make a copy of the ConfigMap for your {RHDHShort} Operator, `backstage-default-config`.
++
+----
+$ oc get configmap backstage-default-config -n <rhdh-operator-namespace> -o yaml > <CopyOfRhdhOperatorConfig>
+----
++
+Replace `<rhdh-operator-namespace>` with your {RHDHShort} Operator namespace, and `<CopyOfRhdhOperatorConfig>` with
```
Comment (g-murray) [inline downstream/modules/devtools/proc-rhdh-backup-operator-configmap.adoc:24]: Totally fine as is, but if considering this last piece is also an instruction you could have it as the 3rd step in this procedure?

File: downstream/modules/devtools/proc-rhdh-backup-operator-configmap.adoc [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+the filename you want to use for your copy of the {RHDHShort} Operator.
+
+
+.Verification
```
Comment (g-murray) [inline downstream/modules/devtools/proc-rhdh-backup-operator-configmap.adoc:28]: Should this have "//" too? Oddly enough it doesn't show in the preview, but according to GH its not commented out?

**Review by g-murray — APPROVED** [review]
LGTM, just some optional callouts.

File: downstream/modules/devtools/proc-rhdh-backup-operator-configmap.adoc (hunk 1 above)
Comment (ariordan-redhat) [reply downstream/modules/devtools/proc-rhdh-backup-operator-configmap.adoc:24]: Well it's kind of an instruction about replacing content in the command, so I thought it should be next to the command. I'll move it to before the command.

**Review by annibond — APPROVED** [review]
lgtm!

**Pull request merged**